src/problem_depot/problem_depot.jl

```diff
@@ -1,5 +1,7 @@
-# Some code in `src/problem_depot` was modified from MathOptInterface
-# which is available under an MIT license (see LICENSE).
```

Comment:
I've removed the reference to MathOptInterface's MOI. Looking through this file, I don't see anything that justifies it. I don't think using a constant to store a benchmark suite justifies it.